web/impact/impact/v1/helpers/criterion_option_spec_helper.py

```diff
@@ -44,6 +45,33 @@ class CriterionOptionSpecHelper(ModelHelper):
     ]
     INPUT_KEYS = ALL_KEYS
 
+    def __init__(self, subject):
+        super().__init__(subject)
+        criterion = self.subject.criterion
+        self.criterion_helper = CriterionHelper.find_helper(
+            criterion.type, criterion.name)(subject)
+
+    def app_ids_for_feedback(self, feedbacks, option_name, applications):
+        return self.criterion_helper.app_ids_for_feedback(
+            feedbacks, option_name=option_name, applications=applications)
+
+    def options(self, apps):
+        return self.criterion_helper.options(self.subject, apps)
+
+    def app_count(self, apps, option_name):
+        return self.criterion_helper.app_count(apps, option_name)
+
+    def total_commitments(self, commitments, option_name):
```

Comment:
likewise here, should be `total_capacity` and below `remaining_capacity`

Reply:
"